doc/openocd.texi

```diff
@@ -11369,16 +11369,28 @@ OpenOCD. When off, they generate a breakpoint exception handled internally.
 The commands below can be used to prevent OpenOCD from using certain RISC-V trigger features.
 For example in cases when there are known issues in the target hardware.
 
-@deffn {Command} {riscv set_enable_eq_match_trigger} [on|off]
-When on (default), allow OpenOCD to use exact-match triggers in watchpoints.
-@end deffn
+@deffn {Command} {riscv set_enable_trigger_feature} [(@option{eq}|@option{napot}|@option{ge_lt}|@option{all}) (@option{wp}|@option{none})]
+Control which RISC-V trigger features can be used by OpenOCD placing watchpoints.
+All trigger features are allowed by default. Only new watchpoints, inserted after this command,
+are affected (watchpoints that were already placed before are not changed).
```

Comment:
```suggestion
All trigger features are allowed by default. Only new triggers, inserted after this command,
are affected (triggers that were already placed before are not changed).
```

Reply:
I would like to keep the current version, because it informs user (one more time) that this command affects only on watchpoints.